src/capture_RPiHQ.cpp

```diff
@@ -1014,16 +1129,29 @@ int main(int argc, char *argv[])
 
 	if (help == 1)
 	{
+		printf("%sUsage:\n", c(KRED));
+		printf(" ./capture_RPiHQ -width 640 -height 480 -nightexposure 5000000 -gamma 50 -nightbin 1 -filename Lake-Laberge.JPG\n\n");
+		printf("%s", c(KNRM));
+
 		printf("%sAvailable Arguments:\n", c(KYEL));
+		printf(" -cmd                               - Default = raspistill - cmd being used\n");
 		printf(" -width                             - Default = Camera Max Width\n");
 		printf(" -height                            - Default = Camera Max Height\n");
 		printf(" -nightexposure                     - Default = 5000000 - Time in us (equals to 5 sec)\n");
 		printf(" -nightautoexposure                 - Default = 0 - Set to 1 to enable auto Exposure\n");
 		printf(" -autofocus                         - Default = 0 - Set to 1 to enable auto Focus\n");
 		printf(" -nightgain                         - Default = 4.0 (1.0 - 16.0)\n");
 		printf(" -nightautogain                     - Default = 0 - Set to 1 to enable auto Gain at night\n");
-		printf(" -saturation                        - Default = 0 (-100 to 100)\n");
-		printf(" -brightness                        - Default = 50 (0 to 100)\n");
+		if (is_libcamera)
```

Comment:
I think we have to inform the user in readme/wiki. After update (to bullseye) he has to redefine some settings. But maybe it is useless, because he has to make a complete new setup.

Reply:
Thanks for the reminder.
I added this to the "settings" Wiki page:

**IMPORTANT NOTE FOR PEOPLE WITH AN RPiHQ CAMERA ON THE "BULLSEYE" OPERATING SYSTEM**

Several settings have different ranges on Bullseye versus Buster, so you'll need to update them. For example, "Brightness" in Buster ranged from 0 (black) to 100 (white). In Bullseye it's now -100 to 100.